include/tank.h

```diff
@@ -0,0 +1,83 @@
+/**
+ * @file tank.h
+ */
+#ifndef TANK_H
+#define TANK_H
+
+#include <cm.h>
+#include <po.h>
+#include <joy.h>
+#include <zap.h>
+#include <step.h>
+#include <flash.h>
+
+// Forward.
+enum MSGID;
+
+/**
+ * @brief Unknown.
+ * @todo Move elsewhere?
+ */
+enum JTHS
+{
+    JTHS_Nil = -1,
+    JTHS_Normal = 0,
+    JTHS_Damaged = 1,
+    JTHS_Dead = 2,
+    JTHS_Max = 3
+};
+
+/**
+ * @brief Tank State (?)
```

Comment:
Struct Name + Capital S is almost always the state enum for that struct, so this would be Tank State.